e2e/providerconfig-aws-e2e-test/Makefile

```diff
@@ -0,0 +1,106 @@
+# Project Setup
+PROJECT_NAME := providerconfig-aws-e2e-test
+PROJECT_REPO := github.com/upbound/provider-upjet-aws
+
+# NOTE(hasheddan): the platform is insignificant here as Configuration package
+# images are not architecture-specific. We constrain to one platform to avoid
+# needlessly pushing a multi-arch image.
+PLATFORMS ?= linux_amd64
+-include ../../build/makelib/common.mk
+
+# ====================================================================================
+# Setup Kubernetes tools
+
+KIND_VERSION = v0.22.0
+UP_VERSION = v0.28.0
+UP_CHANNEL = stable
+UPTEST_VERSION = v0.11.1
```

Comment:
Please upgrade to the latest one `v0.13.1`. Also, now we are using the differente source for downloading the uptest artifacts. Please see the `UPTEST_LOCAL` in the root Makefile.